components/nimbus/src/enrollment.rs

```diff
@@ -900,18 +886,18 @@ impl<'a> EnrollmentsEvolver<'a> {
             false
         };
 
-        let mut targeting_attributes = self.targeting_attributes.clone();
-        targeting_attributes.is_already_enrolled = is_already_enrolled;
+        let th = self
+            .targeting_helper
+            .put("is_already_enrolled", is_already_enrolled);
```

Comment:
This step is a bit icky, and should be removed, but we need to change `experimenter` sticky logic to use the `active_experiments` form of sticky targeting.

Reply:
Yes, agreed with caveats: we lose the type-safety within the evolver: so I'm leaning towards getting rid of mutability of the targeting helper (mutability/clone with changes that we have here) within the evolver. In practice, I think this means changing just this sticky flag.

Reply:
Created https://github.com/mozilla/experimenter/issues/8661